azure-pipelines.yml

```diff
@@ -0,0 +1,57 @@
+jobs:
+
+- job:
+  strategy:
+    matrix:
+      Mojave-Xcode-10.2.1:
+        IMAGE_POOL: 'macOS-10.14'
+        XCODE_VERSION: '10.2.1'
+      Mojave-Xcode-11.0:
+        IMAGE_POOL: 'macOS-10.14'
+        XCODE_VERSION: '11'
+  pool:
+    vmImage: $(IMAGE_POOL)
+  
+  steps:
+
+  - script: sudo xcode-select --switch "/Applications/Xcode_$(XCODE_VERSION).app/Contents/Developer"
+    displayName: "Select Xcode $(XCODE_VERSION)"
```

Comment:
Let's print the Xcode version and some information about the Xcode environment in this step.

```
script: |
  set -e
  sudo xcode-select ...
  echo "Xcode version: $(xcrun xcodebuild -version)"
  echo "Xcode path: $(xcrun -f xcodebuild)"
  echo "Xcode select path: $(xcode-select --print-path)"
```